.gitignore

```diff
@@ -195,4 +195,9 @@ data/
 test_data/test_outputs
 /env/
 /.quarto/
-/_site/
\ No newline at end of file
+/_site/
+*.PNG
+*.png
+*.html
+*.geojson
+*.pkl
```

Comment:
Added .pkl files to .gitignore to avoid accidentally committing model files. 

These can be bypassed if we're sure we really, really want to. 

But given @butchtm and team's plan of having a config file indicating the location of the model and other relevant files (features, rollout results, etc), think it would be best not to commit model files here anymore and just lump together all these big files somewhere else. The config file should already link the code version and corresponding files appropriately.